sdk/ai/azure-ai-generative/azure/ai/generative/index/_mlindex.py

```diff
@@ -626,21 +732,50 @@ def from_embeddings_container(
                 index_config = {
                     **index_config,
                     **{
-                        "endpoint": index_connection.target if hasattr(index_connection, "target") else index_connection["properties"]["target"],
-                        "api_version": index_connection.metadata.get("apiVersion", "2023-07-01-preview") if hasattr(index_connection, "metadata") else index_connection["properties"]["metadata"].get("apiVersion", "2023-07-01-preview"),
-                    }
+                        "endpoint": index_connection.target
+                        if hasattr(index_connection, "target")
+                        else index_connection["properties"]["target"],
+                        "api_version": index_connection.metadata.get("apiVersion", "2023-07-01-preview")
+                        if hasattr(index_connection, "metadata")
+                        else index_connection["properties"]["metadata"].get("apiVersion", "2023-07-01-preview"),
+                    },
+                }
+                connection_args = {
+                    "connection_type": "workspace_connection",
+                    "connection": {"id": get_id_from_connection(index_connection)},
+                }
+
+            mlindex = create_index_from_raw_embeddings(
+                embeddings,
+                index_config,
+                connection=connection_args,
+                output_path=str(output_path),
+                credential=credential,
+            )
+        elif index_type == "pinecone":
+            from azureml.rag.tasks.update_pinecone import create_index_from_raw_embeddings
```

Comment:
This needs to import from azure.ai.generative: `azure.ai.generative.index._.tasks.update_pinecone`, also I don't think this task has been synced to azure-ai-generative yet, @datdo-msft is planning to do that tomorrow, so for now maybe leave our handling of pinecone indexes?

Reply:
I have to addi entire update_pinecone.py into this PR.